apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -63,6 +64,14 @@ expression: "&schema"
           },
           "nullable": true
         },
+        "max_age": {
+          "description": "The `Access-Control-Max-Age` header value in seconds",
+          "default": null,
+          "type": "integer",
```

Comment:
I think elsewhere in our configuration, the pattern is for this to accept a human-friendly value that more clearly lets the reader understand the unit of measurement: e.g., accepting `5m` for 5 minutes or `3600s` for 3600 seconds.

Choosing to merely accept seconds as an integer would deviate from that convention.  WDYT?

Reply:
Example here https://github.com/apollographql/router/blob/bnjjj/fix_2406/apollo-router/src/plugins/authentication.rs#L80:L80

Reply:
I think that's fair, I can take a shot at this and also add the necessary documentation

Reply:
Addressed!